### PR TITLE
feature/TECH 694 Improve traefik config

### PR DESCRIPTION
**Pull request by koeves**
- [x] Create HTTP ingress routes that redirect to the HTTPS one.
- [x] Add priority for routes
- [x] Add insecure option 
- [x] Set both maintainer and maintainers fields in the metadata
- [x] Use "service" as the default name of containers

----
### 📕 [TECH-694](https://vandebron.atlassian.net/browse/TECH-694) Add HTTP routes danielkoves@vandebron.nl 

🏗️ Build [3](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-299/3/display/redirect) ✅ Successful, started by _Daniel Koves_  
🚀 *[cloudfront-service](https://cloudfront-service-299.test.nl/swagger/index.html)*, *example-dagster-user-code*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-299.test.nl/swagger/index.html)*, *ocpp-first*, *ocpp-second*, *[sbtservice](https://sbtservice-299.test.nl/swagger/index.html)*, *sparkJob*  

[TECH-694]: https://vandebron.atlassian.net/browse/TECH-694?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Jorg88 — APPROVED** [review]

File: src/mpyl/project.py [hunk 1]
```diff
@@ -345,6 +345,8 @@ class TraefikHost:
     service_port: Optional[int]
     tls: Optional[TargetProperty[str]]
     whitelists: TargetProperty[list[str]]
+    priority: Optional[int]
+    insecure: bool = False
```
Comment (Jorg88) [inline src/mpyl/project.py:349]: nit: seems a bit redundant to set a default value twice?

File: src/mpyl/steps/deploy/k8s/chart.py [hunk 2]
```diff
@@ -241,11 +241,10 @@ def _to_labels(self) -> dict:
             "app.kubernetes.io/instance": self.release_name,
         }
 
-        if len(self.project.maintainer) > 1:
+        if len(self.project.maintainer) > 0:
```
Comment (Jorg88) [inline src/mpyl/steps/deploy/k8s/chart.py:244]: why is this? seems a bit ugly

File: src/mpyl/steps/deploy/k8s/chart.py [hunk 3]
```diff
@@ -714,7 +716,7 @@ def to_deployment(self) -> V1Deployment:
         liveness_probe, startup_probe = self._construct_probes()
 
         container = V1Container(
-            name=self.release_name,
+            name="service",
```
Comment (Jorg88) [inline src/mpyl/steps/deploy/k8s/chart.py:719]: and this?

File: src/mpyl/steps/deploy/k8s/chart.py (hunk 2 above)
Comment (koeves) [reply src/mpyl/steps/deploy/k8s/chart.py:244]: apparently it's on purpose

Comment (koeves) [reply src/mpyl/steps/deploy/k8s/chart.py:244]: talked to Tim about this

File: src/mpyl/steps/deploy/k8s/chart.py (hunk 3 above)
Comment (koeves) [reply src/mpyl/steps/deploy/k8s/chart.py:719]: same

Comment (koeves) [reply src/mpyl/steps/deploy/k8s/chart.py:719]: both of them were like this in MPL, this particular change helps with querying the logs of the service

**Pull request merged**